pfclient/pfclient.go

```diff
@@ -348,29 +262,21 @@ func (p *pfclient) MarkContainerAsDeleted(node string, hostname string) (bool, e
 	return true, nil
 }
 
-func (p *pfclient) StoreMetrics(metrics *pfmodel.Metrics) (bool, error) {
-	// Setup address and query params
-	addr := fmt.Sprintf("%s/%s", p.pfServerAddr, p.pfApiPath["StoreMetrics"])
+func (p *pfclient) UpdateContainerStatus(node, hostname, status string) (bool, error) {
```

Comment:
should we open this API? I think this should be keep as private function @fadlinurhasan

Reply:
Updated Mas.